test/truefi2/TrueAssuranceFund.test.ts

```diff
@@ -28,4 +39,13 @@ describe('TrueAssuranceFund', () => {
     await safu.liquidate(loan.address)
     expect(await token.balanceOf(safu.address)).to.equal(0)
   })
+
+  it('fails if loan is not defaulted', async () => {
+    await expect(safu.liquidate(loan.address)).to.be.revertedWith('TrueAssuranceFund: Loan is not defaulted')
+  })
+
+  it('fails if loan is not defaulted', async () => {
```

Comment:
Let's update this description (looks like you might've just copy/pasted the previous test description and forgotten to rename it).

Reply:
😅